entries/QUnit.config.xml

```diff
@@ -35,6 +35,13 @@
 				<p>Prior to version 1.16.0, this property was called <code>module</code>.</p>
 			</desc>
 		</property>
+		<property name="seed" type="String" default="undefined">
+			<desc>
+				This property tells QUnit to run tests in a seeded-random order. The value provided will be used as the seed in a pseudo-random number generator to ensure that results are reproducible.
```

Comment:
We should document that this property respects `reorder` (i.e., that previously-failing tests are still executed first).